src/main/java/array/Exer09.java

```diff
@@ -0,0 +1,32 @@
+package array;
+
+public class Exer09 {
+
+    public int binarySearch(int[] array, int n) {
+        int start = 0;
+        int end = array.length;
+        int middle;
+
+        while (end - start > 1) {
```

Comment:
Em chưa thực sự hiểu về thuật toán nên nó mới ra rắc rối như thế này. Cố gắng đọc rồi làm lại nha

Reply:
Dạ cảm hơn anh.
Nhận xét cái nào là không sai đi đâu được :))

Reply:
fixed